ivy/functional/frontends/torch/tensor.py

```diff
@@ -380,6 +380,12 @@ def not_equal(self, other, *, out=None):
 
     ne = not_equal
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
+    def equal(self, other, *, out=None):
```

Comment:
equal function don't take this arguments 😄  please check the torch docs and change it after that be sure that all tests is passing